### ISSUE TITLE
PROJECT khm18

### ISSUE THREAD

**Issue by khm18** [wonsunahn/Capstone_Spring2020#30]
1. Privatizers for the Internet of Things - measuring the impact of privatizer functions on image detection models
-I have taken a class on Computer Vision and have some experience and insight as to how recognition works. However, I do not know too much of TensorFlow or how to use it. I find the concept of using privatizer functions to be very interesting and wish to be able to learn more about it! I am able to program in Python but not as efficiently as Java, and I have a little experience with applying Neural Networks to give labels to images and such. 

2. Behaivior - Develop a Mobile Application to Help Support People in Recovery from Opioid Addiction
-I have little experience in mobile application development, but this project stood out to me. To be able to create a tangible application that can potentially help people going through addiction is what pulled me in! I have experience in Java, Python, and a tiny bit using Android Studios, but I am willing to learn what is necessary for me!

3. Create a Voting System for SCI
-I have some experience in Web Development, both front-end, and back-end. I have taken a class to understand more of the intricacies of what web development really is, and I also wish to understand more real-life applications of such a skillset and toolset. I have experience in Javascript, HTML, CSS, SQLite, and Python. I am also willing to learn more if necessary!

4. Remote veterinary care app
-To be able to help give antibiotic prescriptions to those who do not have ease of access is what pulled me into this. The concept of being able to help someone out is a driving factor of why I am interested in such a topic! I don't have too much experience in application development, but I am willing to learn!

5. A website for handing-in programming assignments
-I have some experience with front-end and back-end web development and have taken a class earlier to understand more of what is going on. Although this isn't my first choice, I still have some keen interest in being able to develop a service that is capable of making lives more convenient!
